src/components/tab/tab.test.tsx

```diff
@@ -99,4 +107,28 @@ describe('Tab', () => {
       );
     });
   });
+
+  describe('With Badge', () => {
+    it('should render badge', async () => {
+      sut({ ...defaultTab, badge: { label: 2 } });
+      expect(screen.getAllByTestId('ion-badge'));
+    });
+
+    it('should not render badge when is not informed', async () => {
+      sut();
+      expect(screen.queryAllByText('ion-badge')).toHaveLength(0);
```

Comment:
screen.queryAllByText('ion-badge') this is correct? I believe that you did should find by badge label or by testid.